<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea_all_peers">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>40</y>
      <width>200</width>
      <height>751</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollContents_all_peers">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>198</width>
       <height>749</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bütün Peerlar</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_peers_that_i_subscribe">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>40</y>
      <width>200</width>
      <height>750</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollContents_peers_that_i_subscribe">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>198</width>
       <height>748</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Takip Ettiğim Peerlar</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>821</width>
      <height>391</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_subscriber_requests">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>200</width>
      <height>311</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollContent_subscriber_requests">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>198</width>
       <height>309</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Takip İstekleri</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>450</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Engellenen Kullanıcılar</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_peers_that_i_block">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>200</width>
      <height>311</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollContents_peers_that_i_block">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>198</width>
       <height>309</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Beni Engelleyen Kullanıcılar</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_peers_that_block_me">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>480</y>
      <width>200</width>
      <height>311</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollContents_peers_that_block_me">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>198</width>
       <height>309</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Takipçilerim</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_subscribers">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>480</y>
      <width>200</width>
      <height>311</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollContents_subscribers">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>198</width>
       <height>309</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
